tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -615,3 +624,132 @@ def custom_continue_workflow(self, *args, **kwargs):
     assert obj.extra_data['is-update']
     assert obj.extra_data['approved']
     assert obj.status == ObjectStatus.COMPLETED
+
+
+def test_validation_error_callback_valid(workflow_app):
+    invalid_record = {
+        '_collections': [
+            'Literature',
+        ],
+        'document_type': [
+            'article',
+        ],
+        'titles': [
+            {'title': 'A title'},
+        ],
+    }
+
+    eng_uuid = start('article', [invalid_record])
+
+    eng = WorkflowEngine.from_uuid(eng_uuid)
+    obj = eng.objects[0]
+
+    response = do_validation_callback(
+        app=workflow_app,
+        data=obj,
+    )
+
+    assert response.status_code == 200
```

Comment:
Probably you should assert the side effects of the validation acceptance here, to make sure that the workflow is continued and such.